From de1bc9b66ca12b18e2517daf5d11899578c85618 Mon Sep 17 00:00:00 2001
From: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
Date: Mon, 12 Jul 2021 20:46:23 -0500
Subject: [PATCH 1/1] u-boot: Add spi2 entry to device tree enable local spi
 rom flash update
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit

Adds spi2 entry to device tree enable local spi rom flash update.

Signed-off-by: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
---
 arch/arm/dts/ast2600-hawaii.dts | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm/dts/ast2600-hawaii.dts b/arch/arm/dts/ast2600-hawaii.dts
index 83cfa66dd2..94118ee5c8 100644
--- a/arch/arm/dts/ast2600-hawaii.dts
+++ b/arch/arm/dts/ast2600-hawaii.dts
@@ -106,6 +106,21 @@
 	};
 };

+&spi2 {
+	status = "okay";
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_spi2_default &pinctrl_spi2quad_default>;
+
+	flash@0 {
+		compatible = "spi-flash";
+		status = "okay";
+		spi-max-frequency = <50000000>;
+		spi-tx-bus-width = <4>;
+		spi-rx-bus-width = <4>;
+	};
+};
+
 &sdhci {
 	timing-phase = <0xc6ffff>;
 };
--
2.17.1

